atlasdb-client/src/main/java/com/palantir/atlasdb/schema/TargetedSweepSchema.java

```diff
@@ -137,6 +137,25 @@ private static void addSweepStateTables(Schema schema) {
                 conflictHandler(ConflictHandler.IGNORE_ALL);
             }
         });
+
+        schema.addTableDefinition("sweepAssignedBuckets", new TableDefinition() {
```

Comment:
not stuck to the name - this changed multiple times, but something bucket and assignment seemed fitting, as we refer to the process of assigning timestamp ranges to buckets as bucket assignment.